velox/connectors/hive/HiveDataSource.h

```diff
@@ -94,6 +94,10 @@ class HiveDataSource : public DataSource {
 
   static void registerWaveDelegateHook(WaveDelegateHookFunction hook);
 
+  const ConnectorQueryCtx* testingConnectorQueryCtx() const {
```

Comment:
is this pattern of using the "testing" prefix for method only used for testing documented in our coding style doc?

Reply:
Yes, it is documented.